uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlement.php

```diff
@@ -199,12 +199,13 @@ private static function buildLeavePeriodParamsFromCalculation(
     $absenceTypeID = $calculation->getAbsenceType()->id;
     $contactID = $calculation->getContact()['id'];
     $absencePeriodID = $calculation->getAbsencePeriod()->id;
+    $entitlementOverridden = isset($overriddenEntitlement) && trim($overriddenEntitlement) != '';
```

Comment:
Given we need a boolean value, wouldn't this be better? `!empty(trim($overriddenEntitlement))`